src/app/missions/detail/detail.component.css

```diff
@@ -9,6 +9,107 @@
     padding-right: 1.5rem;
 }
 
+.chat-bubble {
+    background-image: url("/assets/ui/chat-bubble.svg");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    border: solid 1px #ea6102;
+    border-radius: 11px;
+    color: black;
+    display: inline-block;
+    font-weight: bold;
+    line-height: 1.1;
+    max-width: 200px;
+    padding: 0.6rem 0.8rem;
+    text-align: center;
+    word-wrap: break-word;
+}
+
+.achievement-tooltip {
+    background-image: url("/lu-res/ui/ingame/achievements_i1e.png");
+    background-size: 130% 175%;
+    display: flex;
+		max-width: 256px;
+    padding: 0.5rem;
+    padding-left: 1rem;
+}
+
+.achievement-tooltip-icon {
+    align-self: center;
+}
+
+.achievement-tooltip-text {
+    align-self: center;
+    background-color: transparent;
+    color: black;
+    font-size: 1.1rem;
+    font-weight: bold;
+		line-height: 1rem;
```

Comment:
`line-height` in root-em seems a bit weird. It looks a bit dense on my screen, but is fine for now.